bin/options.js

```diff
@@ -62,17 +62,6 @@ const options = {
     describe: 'Open default browser with the specified page',
     requiresArg: true,
   },
-  color: {
-    type: 'boolean',
-    alias: 'colors',
-    default: function supportsColor() {
-      // Use `require('supports-color').stdout` for supports-color >= 5.0.0.
-      // See https://github.com/webpack/webpack-dev-server/pull/1555.
-      return require('supports-color').stdout;
```

Comment:
Do not forget to remove `supports-color`

Reply:
Done here: https://github.com/webpack/webpack-dev-server/pull/2660/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L62